dependency-reduced-pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.yanirta</groupId>
   <artifactId>ImageTester</artifactId>
-  <version>2.3.5</version>
+  <version>3.0.4</version>

```

Comment:
You can remove this file, it's generated by the maven-shade-plugin. 

I should have added it to the .gitignore file.